.dockerignore

```diff
@@ -7,7 +7,6 @@ coverage/
 Dockerfile
 
 .gitignore
-.env
```

Comment:
Actually, the `.env` file should not be in the docker image. The proper way to use environment variables in docker is documented here: https://docs.docker.com/compose/environment-variables/